zerver/tests/test_subs.py

```diff
@@ -6746,7 +6804,7 @@ def test_gather_subscriptions(self) -> None:
         for user in [cordelia, othello, polonius]:
             self.assert_user_got_subscription_notification(user, msg)
 
-        with self.assert_database_query_count(7):
+        with self.assert_database_query_count(9):
```

Comment:
Haven't looked too deep into this, but I don't think we can avoid this, we will have to fetch recursive user group ids. Same applies to other query count increases in `test_subs`